network/identity.go

```diff
@@ -117,13 +128,37 @@ func (i *identity) start() error {
 	return nil
 }
 
-func (i *identity) getStatus() *proto.Status {
-	return &proto.Status{
-		Chain: int64(i.srv.config.Chain.Params.ChainID),
+func (i *identity) getStatus(peerID peer.ID) *proto.Status {
```

Comment:
Method is called `getStatus`, but you initialize the status response here

Reply:
renamed it, commit-[link](https://github.com/0xPolygon/polygon-edge/pull/343/commits/d3a3b8c0751b8d49287ec786a62eec524f3faef6)

Reply:
The method is called `currentStatus`, but you initialize the status inside the method. A more appropriate name here is `constructStatus` or something similar

cc @Kourin1996

Reply:
Done @zivkovicmilos  commit-[link](https://github.com/0xPolygon/polygon-edge/pull/343/commits/cdbb38680ac0aa125d2d7e856028ecaab55bfd46)